examples/README.md

```diff
@@ -57,6 +61,11 @@ spec:
 Note: Specyfing an empty string for the `harborAdminPassword`-key in `spec.helmChart.valuesYaml` will trigger password generation by the harbor instance itself.
 The admin password will then be saved under the key `HARBOR_ADMIN_PASSWORD` in a secret named `HELM_RELEASE_NAME`-`harbor-core`.
 
+[Harbor Garbage Collection](https://goharbor.io/docs/1.10/administration/garbage-collection/) can be configured via `spec.garbageCollection`.
+Valid values for `.spec.garbageCollection.schedule.type` are `Hourly`, `Daily`, `Weekly`, `Custom`, `Manual`, and `None` (each starting with a capital letter).
```

Comment:
So... What is `garbageCollection.schedule.cron` even good for? Should we even expose it at all?

Reply:
This **has** to be set to a cron expression. The Harbor API will return an error when leaving this out, sadly. Im going to add comments on this, here.